.buildkite/pipeline.yml

```diff
@@ -19,6 +19,9 @@ steps:
   # multiple go builds are downloading to the same cache.
   - wait
 
+  - label: 'git log validation'
```

Comment:
Can you move this above the `- wait` step so it runs in parallel with `go mod download`?  Or did you put it here so that failure in the log validation doesn't block tests from running?

Reply:
Yeah, I put that after the wait because I want to run tests on the code as it exists, even if the commit log isn't pretty.

Reply:
:+1: It'd be worth adding a comment I think.

Reply:
Added a comment.